### DISCUSSION TITLE
Do Popover's have a hook for when they are closed?

### DISCUSSION THREAD

**Discussion by itsjms** [tailwindlabs/headlessui#1422] [Help]
Using Vue (not that it really matters)

Do Popovers not have some sort of `@close` event they emit like Dialog's do?

How can you tell when the user closes a Popover by clicking anywhere outside of the Popover without being "hacky"?

**Comment by waqas-ali-pk** [comment]
@itsjms Have you found some workaround for this?

**Reply by itsjms** [reply]
Nope! I just worked around it for now. You could probably wrap it in another component with a default slot and do some state tracking within that or something.
